src/helpers/decodeToken.js

```diff
@@ -0,0 +1,11 @@
+import jwt from 'jsonwebtoken';
```

Comment:
please get rid of all this.

The worst you could do is:

stringify the values from the backend and store the values in the localstorage.

but don't decode the token in the frontend

Reply:
you could also send back the `expires in` part from the backend if you want (raise a ticket for that)